tox.ini

```diff
@@ -49,19 +51,31 @@ commands =
     pre-commit run --all-files
 
 
-[testenv:pre-merge-{all,ano,cls,det,seg,act}-{py38,py39,py310}]
+[testenv:pre-merge-{all,ano,cls,det,iseg,seg,act}-{py38,py39,py310}]
 deps =
     {[testenv]deps}
     -r{toxinidir}/requirements/dev.txt
 use_develop = true
 commands =
     coverage erase
-    coverage run -m pytest -ra --showlocals --junitxml={toxworkdir}/test-results.xml {posargs:tests/integration/{[testenv]test_dir}}
+    coverage run -m pytest -ra --showlocals --junitxml={toxworkdir}/test-results-{envname}.xml {posargs:tests/integration/{[testenv]test_dir}}
     coverage report -m --fail-under=0
     coverage xml -o {toxworkdir}/coverage.xml
 
 
-[testenv:fuzzing-test]
+[testenv:tests-{all,ano,cls,det,iseg,seg,act}-{py38,py39,py310}]
+deps =
+    {[testenv]deps}
+    -r{toxinidir}/requirements/dev.txt
+use_develop = true
+commands =
+    coverage erase
+    coverage run -m pytest -ra --showlocals --junitxml={toxworkdir}/{envname}.xml {posargs:tests/integration/{[testenv]test_dir}}
+    coverage report -m --fail-under=0
+    coverage xml -o {toxworkdir}/coverage.xml
+
+
+[testenv:fuzzing]
```

Comment:
Do we need both?

Reply:
oh... the old one should be removed. thanks.